tests/foreman/cli/test_provisioningtemplate.py

```diff
@@ -197,80 +153,41 @@ def test_positive_create_locked():
 
     :CaseImportance: Medium
     """
-    new_template = make_template({'locked': 'true', 'name': gen_string('alpha')})
+    new_template = module_target_sat.cli_factory.make_template(
+        {'locked': 'true', 'name': gen_string('alpha')}
+    )
     assert new_template['locked'] == 'yes'
 
 
-@pytest.mark.tier2
-def test_positive_create_with_org(module_org):
-    """Check if Template with Organization can be created
-
-    :id: 5de5ca76-1a39-46ac-8dd4-5d41b4b49076
-
-    :expectedresults: Template is created and new Organization has been
-        assigned
-
-    :CaseImportance: Medium
-    """
-    new_template = make_template({'name': gen_string('alpha'), 'organization-ids': module_org.id})
-    assert module_org.name in new_template['organizations']
-
-
 @pytest.mark.tier2
 @pytest.mark.upgrade
-def test_positive_add_os_by_id(module_os_with_minor):
-    """Check if operating system can be added to a template
+def test_positive_add_remove_os_by_id(module_target_sat, module_os_with_minor):
+    """Check if operating system can be added and removed to a template
 
     :id: d9f481b3-9757-4208-b451-baf4792d4d70
 
-    :expectedresults: Operating system is added to the template
+    :expectedresults: Operating system is added/removed from the template
 
     :CaseLevel: Integration
     """
-    new_template = make_template()
-    Template.add_operatingsystem(
-        {'id': new_template['id'], 'operatingsystem-id': module_os_with_minor.id}
-    )
-    new_template = Template.info({'id': new_template['id']})
-    os_string = (
-        f'{module_os_with_minor.name} {module_os_with_minor.major}.{module_os_with_minor.minor}'
+    os = module_os_with_minor
+    os_string = f'{os.name} {os.major}.{os.minor}'
```

Comment:
```suggestion
    os_version
```

Reply:
its just a string, os_version confuses here as it refers to major.minor, so would keep it as is